tests/guardrails/correction_range_schedule.tests.js

```diff
@@ -0,0 +1,111 @@
+const { limits } = require('../../src/index');
+
+var correctionRangeScheduleTests = (test) => {
+    it('cannot set above max', async () => {
```

Comment:
[nit] Can these tests take the same form as above tests where there is one "open" test and then a bunch of individual tests, and then a "close" test?